homeworks/nemethmate/1_introduction/2_vars_datatypes/info.py

```diff
@@ -0,0 +1,66 @@
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+# Programozási nyelvek bekérése
+programming_languages = input("Adj meg 4 programozási nyelvet vesszővel elválasztva, szóközök nélkül: ")
+
+# String konvertálása listává
+skills_list = programming_languages.split(",")
+
+# Skills lista hozzáadása
+user_info["skills"] = skills_list
+
+# A 'favourite_meals' lista rendezése abc sorrendbe
+user_info["favourite_meals"].sort()
+
+# Print az utolsó előtti favourite meal-ről
+
+print(user_info["favourite_meals"][-2])
+
+# Add spaghetti
+
+user_info["favourite_meals"].append("spaghetti")
+
+# A harmadik és negyedik elem hozzáadása mégegyszer
+user_info["favourite_meals"].extend([user_info["favourite_meals"][2], user_info["favourite_meals"][3]])
+
+# Dupla elemek eltávolítása
+user_info["favourite_meals"] = list(dict.fromkeys(user_info["favourite_meals"]))
```

Comment:
szuper megoldás a dict.fromkeys(), nem kérem h változtasd meg, csak mutatok egy másik megoldást is (a set nem enged meg duplikátumokat):
list(set(user_info["favourite_meals"]))

Reply:
Köszi igen. Gondoltam, hogy ez lenne a magától értetődő megoldás csak azt olvastam, hogy a set nem feltétlen őrzi meg az elemek sorrendjét a listában. Lehet fals info volt de emiatt kerestem ezt az alternatív utat.